src/logger.config.ts

```diff
@@ -1,7 +1,10 @@
-import {NgxLoggerLevel} from './types/logger-lever.enum';
+import {NgxLoggerLevel} from './types/logger-level.enum';
+import {Type} from '@angular/core';
+import {NGXLoggerSaveService} from './types/save.service.abstract';
 
 export class LoggerConfig {
   level: NgxLoggerLevel;
-  serverLogLevel?: NgxLoggerLevel;
+  saveLogLevel?: NgxLoggerLevel;
```

Comment:
The reason I changed this is to better server the abstraction. We are saving the log file but not necessarily to the server. I think tying the functionality to the server is too implementation specific. 
I understand that it would be a breaking change and it would require a version bump that allows breaking changes.

Reply:
This is actually a good idea, I was thinking about allowing the user to send ngrx state with logs when making support issue in the UI.